### PR TITLE
main develop refactor branch ruleset json files

### PR DESCRIPTION
**Pull request by Intizar-T**

### CODE REVIEW

File: .github/rulesets/syb_develop_ruleset.json [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{
+  "id": 2539891,
+  "name": "syb_develop_ruleset",
+  "target": "branch",
+  "source_type": "Repository",
+  "source": "tokamak-network/tokamak-sybil-resistance",
```
Comment (kaiden-araki) [inline .github/rulesets/syb_develop_ruleset.json]: @Intizar-T Quick confirm, is it applying to mvp, too?

File: .github/rulesets/syb_develop_ruleset.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        "required_approving_review_count": 1,
+        "dismiss_stale_reviews_on_push": true,
+        "require_code_owner_review": true,
+        "require_last_push_approval": false,
+        "required_review_thread_resolution": true,
+        "automatic_copilot_code_review_enabled": false
+      }
+    }
+  ],
+  "bypass_actors": []
```
Comment (kaiden-araki) [inline .github/rulesets/syb_develop_ruleset.json:35]: What is the intention of `bypass_actors` with empty array for all 3 branches?

File: .github/rulesets/syb_develop_ruleset.json (hunk 1 above)
Comment (Intizar-T) [reply .github/rulesets/syb_develop_ruleset.json]: oh yes okay that's a good point, I'll update this. The rulesets are already applying to `mvp`. I'm including these files in the repo for future references

File: .github/rulesets/syb_develop_ruleset.json (hunk 2 above)
Comment (Intizar-T) [reply .github/rulesets/syb_develop_ruleset.json:35]: we can include users who can bypass these rulesets. But for that to work, the user who adds bypass_actor has to be at least `admin` user. So, in short, this is not a security threat at all, unless the attack is coming from one of the `admins` haha

**Review by kaiden-araki — APPROVED** [review]

**Pull request merged**